### PR TITLE
CAMEL-9180 - camel-cmis Lazy Initialize Session Facade

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CMISConsumer.java [hunk 1]
```diff
@@ -60,4 +58,15 @@ int sendExchangeWithPropsAndBody(Map<String, Object> properties, InputStream inp
...
         return 1;
     }
+
+    private CMISSessionFacade getSessionFacade() throws Exception {
+        if (sessionFacade == null) {
+            sessionFacade = sessionFacadeFactory.create();
+            sessionFacade.initSession();
+        }
+
+        LOG.info("getSessionFacade {}", sessionFacade);
```
Comment (davsclaus) [inline components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CMISConsumer.java]: Reduce the logging level to debug or trace

**Comment by lburgazzoli** [comment]
It should not be there at all, I will remove it

**Comment by lburgazzoli** [comment]
Fixed

**Comment by lburgazzoli** [comment]
merged right ?

**Comment by davsclaus** [comment]
yes

**Pull request closed without merging**